components/Timeline/TimelineAnimateImage.tsx

```diff
@@ -29,16 +29,12 @@ export const TimelineAnimateImage = ({
         className,
       )}
     >
-      <motion.div
-        initial={{ rotateY: 0, opacity: 0 }}
-        whileInView={{
-          rotateY: trapezoidAngle === "left" ? 15 : -15,
-          opacity: 1,
-        }}
-        viewport={{ once: true, amount: 0.3 }} // Trigger animation when 30% of it is visible
-        transition={{ delay: 1.5, duration: 2, ease: "easeInOut" }}
+      <AnimateInView
```

Comment:
@rebeccahongsf Please feel free to add more props to AnimateInView however you like, eg., add an easing prop 😄